app/habit/habit-page.tsx

```diff
@@ -39,6 +40,11 @@ export default function HabitPage({
           Habits
         </h1>
         <div className="flex h-7">
+          <Badge variant="secondary" className="mt-2 ml-1">
+            {!isLoading && data!.length}
```

Comment:
This is conditional rendering, check [docs](https://react.dev/learn/conditional-rendering#logical-and-operator-)